README.md

```diff
@@ -11,7 +11,7 @@ A remote CLI tool for [Mattermost](https://github.com/mattermost/mattermost-serv
 To install the project in your `$GOPATH`, simply run:
 
 ```
-go get -u github.com/mattermost/mmctl
+go get github.com/mattermost/mmctl
```

Comment:
Binaries are now recommended to be installed via `go install`. `go get` for binaries will be deprecated in a future version.

I'd suggest `go install github.com/mattermost/mmctl@latest`

Reply:
Sounds good 👍